asf_core_data/pipeline/preprocessing/preprocess_epc_data.py

```diff
@@ -125,6 +133,7 @@ def preprocess_data(
                 data_path / base_config.PREPROC_EPC_DATA_DEDUPL_PATH,
                 data_path=data_path,
                 batch=batch,
+                check_folder="output",
```

Comment:
`outputs`?

Reply:
I mistyped it so many times that I allowed both `output` and `outputs` as valid values for check_folder.  ;-) But I'll change it to outputs here to avoid confusion.